DI-435/README.md

```
@@ -1,43 +1,84 @@
+    --assay "*TWE38" \
+    --outfile_prefix TWE38 \
+    --run_mode find_qc \
+    --start 2024-03-15 \
+    --end 2024-08-04
+```
 
-**Searching Dates**
-These dates restrict the projects collated to only GRCh38 projects
-which have corresponding GRCh37 projects which were created within the specified dates.
+Example command to find Medicover VCFs with no QC status files:
+```bash
```

Markdownlint (0.35.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)